### PR TITLE
Update script to support API v2 in CUSTOMIZED_SDK_URL

### PR DESCRIPTION
**Pull request by llxia**
- add double quotes
- remove wget (simplify the code to use only curl)
- merge multiple if statements into one
- add echo messages in get.sh
- change maketest.sh permission to 755
- remove unnecessary chmod
- fix node label for zos

Issue: #490
Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
Tested:
- set CUSTOMIZED_SDK_URL to v2 api: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox/456/
- set CUSTOMIZED_SDK_URL to archive SDK url from upstream build: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox/457/
- set SDK_RESOURCE to nightly: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox/458/
- tested on zos internally (passed download and got error with sha not match which is not related to this PR)

**Comment by llxia** [comment]
A separate PR will be created for updating `download_url="https://api.adoptopenjdk.net/$JVMVERSION/$SDK_RESOURCE/$PLATFORM/latest/binary"` to v2 API.

**Comment by llxia** [comment]
fyi @jdekonin

File: get.sh [hunk 1]
```diff
@@ -98,49 +98,42 @@ parseCommandLineArgs()
...
+		download_url="https://api.adoptopenjdk.net/$JVMVERSION/$SDK_RESOURCE/$PLATFORM/latest/binary"
+	else
+		download_url=""
+		echo "--sdkdir is set to $SDK_RESOURCE. Therefore, skip download jdk binary"
+	fi
+	
+	if  [ "${download_url}" != "" ]; then
+		echo "curl -OLJks "${download_url}""
+		curl -OLJks "${download_url}"
 		if [ $? -ne 0 ]; then
```
Comment (sophia-guo) [inline get.sh]: This check may not work for curl -O options. That is saying when curl -OLJ fails to download SDK the return CODE is still 0.

**Comment by llxia** [comment]
- added check for binary file and tested with invalid url
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox/463
- regular case:
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox/462/

This binary file check only catches invalid url for current release and nightly url which end with binary. I think we should come up with a better solution once nightly and release url is updated to API v2.

**Pull request merged**